howto/manage-clouds.md

```diff
@@ -2,7 +2,29 @@
 # Manage clouds
 > See first: {ref}`cloud`
 
-(manage-an-entitys-relation-to-a-cloud)=
-## Manage an entity's relation to a cloud
+(manage-permissions-related-to-a-cloud)=
+## Manage permissions related to a cloud
 
-See {ref}`manage-relations`.
\ No newline at end of file
+Given an entity A and a cloud, to grant A permissions on the cloud run the `add-permission` command followed by the tag of A, the desired cloud permission, and the tag of the cloud, where the possible (A, permission, cloud) combinations are:
+
+```
+("cloud:some_cloud#administrator", "can_addmodel", "cloud:some_cloud")
```

Comment:
I think this list of types is not useful here, it's super hard to read. I would just get rid of it. An improvement would be to turn it into a more readable table or something similar. You can see for example that the type on the right hand side is always "cloud:" so that bit doesn't need to be repeated.

Reply:
And the same with all the other pages.

Reply:
Fixed.

Reply:
OK.

Reply:
Deleted.